### PR TITLE
Updates on Django and Python support

### PR DESCRIPTION
**Pull request by michael-k**
- Drop Python 3.3 ([EOL](https://www.python.org/dev/peps/pep-0398/#x-end-of-life))
- Drop Django < 1.8 ([EOL over 2 years ago](https://www.djangoproject.com/download/#supported-versions))
- Minor adjustments for Python 3.6 and Django 1.11 (eg. classifiers, text matrix)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,8 +34,6 @@ Or test with `tox` if you have `tox` installed.
 
 [Changelog on Read The Docs](https://django-braces.readthedocs.io/en/latest/changelog.html)
 
-## Use Django 1.4.x?
+## Supported Django Versions
 
-`django-braces` 1.4.x will be the last version to officially support Django 1.4.x. Since Django 1.4.x is an LTS, we'll update `django-braces` 1.4.x as needed for bug fixes but it won't receive new functionality unless backporting is 100% painless.
-
-Our policy going forward is that `django-braces` officially supports the current version of Django and one version each direction (e.g. 1.6.x is current, so 1.5.x, 1.6.x, and 1.7.x are all supported). There won't be any restraints on using other versions of Django, though, but it will be a "buyer beware" situation.
+Our policy is that `django-braces` officially supports the current version of Django and one version each direction (e.g. 1.6.x is current, so 1.5.x, 1.6.x, and 1.7.x are all supported). There won't be any restraints on using other versions of Django, though, but it will be a "buyer beware" situation.
```
Comment (michael-k) [inline README.md]: When 1.6 ist the current release, there is no 1.7.x …
And there are no tests against Django master

**Comment by michael-k** [comment]
Should I also drop support for Django 1.9 and 1.10?

**Comment by michael-k** [comment]
rebased and dropped Django < 1.11

**Comment by michael-k** [comment]
@kennethlove Can you merge this? I'm willing to add support for Python 3.7 and Django 2.1, but not until this is merged.

**Comment by kennethlove** [comment]
Once you get the 3.7 and 2.1 changes in there (thank you, by the way!) I'll cut a release.

I'm planning on putting django-braces into stasis, almost, as it's feature-complete and doesn't need a ton of maintenance aside from keeping up with Django version/changes.

**Pull request merged**